lambda/app/src/keycloak/client.ts

```diff
@@ -24,7 +24,7 @@ const disableClient = async (data: {
 
   const { kcAdminClient } = await getAdminClient({ serviceType, environment });
   const { realm, client } = await getClient(kcAdminClient, { serviceType, realmName, clientId });
-  if (!client) throw Error('client not found');
+  if (!client) return false;
```

Comment:
oh, this resolves the promise before an update would be triggered, took me a second to figure out :)